todo_app/data/card.py

```diff
@@ -0,0 +1,25 @@
+class Card:
+    def __init__(self, json_input, list):
```

Comment:
Not really important at all, but I think a nice chance here to think about this: why does the original suggestion use a separate class method to filter out the json inputs into the actual values? e.g. more like below:
```python
    def __init__(self, id, closed, due, ... ):
       self.id = id
       self.closed = closed
       ...

    @classmethod
    def from_trello_card(cls, json_input, list):
        return cls(
            json_input['id'],
            json_input['closed'],
            json_input['due'],
            json_input['idBoard'],
            ...
        )
```

Partly this is just about mentally compartmentalising _our_ idea of a "Card" from Trello's idea, but it also gives us a bit more inherent flexibility - e.g. if we started to store _some_ cards in another data store (like a database) then we could just add a new classmethod that uses the same constructor, and take advantage of not having to change that (a nice example of the "Open Closed Principle" - Open to extension but closed to modification)